rust_src/src/windows.rs

```diff
@@ -51,10 +52,30 @@ impl LispWindowRef {
         self.contents.is_not_nil()
     }
 
+    // Equivalent to WINDOW_BOTTOMMOST_P
```

Comment:
Should we expose ports of macros like this to C under their original name if they are still used?

Reply:
This is actually done with a few macros in `windows.rs`, I suppose it would make sense to do it with more of them.